docs/swagger/gncs-api.yml

```diff
@@ -155,16 +165,79 @@ components:
           type: string
           description: The Information for the Event
           example: Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
-        prePayment:
+        cover:
+          type: string
```

Comment:
warum is das ein String? Sollte das nicht ein Bild sein?

Reply:
ich dachte da würde man die url angeben

Reply:
= link zum cover. oder attachment